# Author: Damyan Ivanov
# Purpose: add ExtUtils::Miniperl to the list of core modules of 5.10.0
# Debian-Bug: http://bugs.debian.org/508876
# Upstream-Bug: to-be-filed
--- a/lib/Module/CoreList.pm
+++ b/lib/Module/CoreList.pm
@@ -8281,6 +8281,7 @@ for my $version ( sort { $a <=> $b } key
 	'ExtUtils::Installed'   => '1.43',
 	'ExtUtils::Liblist'     => '6.42',
 	'ExtUtils::Liblist::Kid'=> '6.42',
+	'ExtUtils::Miniperl'    => undef,
 	'ExtUtils::MM'          => '6.42',
 	'ExtUtils::MM_AIX'      => '6.42',
 	'ExtUtils::MM_Any'      => '6.42',
